src/test/resources/bags/basic-sequence-unpruned/README.txt

```diff
@@ -0,0 +1,45 @@
+This is a sequence of three bags. First a is added, then b and then c.
```

Comment:
@jo-pol : I am going to merge this one. Please, do a PR for the issues you raised above.